tests/examples/pacer/nef/s3/paeb_2.json

```diff
@@ -16,7 +16,7 @@
           "pacer_doc_id": null,
           "pacer_magic_num": null,
           "pacer_seq_no": null,
-          "short_description": "CHAP - Hearing Continued"
+          "short_description": "CHAP - Hearing Continued (Bk Other)"
```

Comment:
On keeping `"CHAP"` and `"(Bk Other)"`
![image](https://github.com/user-attachments/assets/ffee8906-f1ec-42c0-a4f8-79dd200ddbb7)